### PR TITLE
Run groovylint on top-level gradle files and on all groovy files

### PR DESCRIPTION
**Pull request by ala-ableton**
@mst-ableton, @nre-ableton: please review, thanks!

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ runTheBuilds.runDevToolsProject(
   test: {
     parallel(failFast: false,
       groovylint: {
-        groovylint.check('./Jenkinsfile,./src/**.groovy,*.gradle')
+        groovylint.check('./Jenkinsfile,**/*.gradle,**/*.groovy')
```
Comment (nre-ableton) [inline Jenkinsfile]: I think that `./*.gradle` would be preferred. The other regexp is fine, though.

**Review by nre-ableton — COMMENTED** [review]
Nit: typo in commit message ("groove")

File: Jenkinsfile (hunk 1 above)
Comment (ala-ableton) [reply Jenkinsfile]: If `./*.gradle` is preferred, then I guess we should change https://github.com/AbletonDevTools/postgres-pipeline-utils/blob/master/Jenkinsfile#L14 and https://github.com/AbletonDevTools/python-pipeline-utils/blob/master/Jenkinsfile#L15.

Comment (nre-ableton) [reply Jenkinsfile]: I would prefer that, yes. However, let's double-check and confirm that it works first.

**Comment by ala-ableton** [comment]
@nre-ableton I amended the commit, the commit message and the PR title. Please have another look, thanks!

**Review by nre-ableton — APPROVED** [review]
LGTM

**Pull request merged**